waffle-compiler/src/compileSolcjs.ts

```diff
@@ -50,7 +52,7 @@ async function resolveSemverVersion(version: string) {
   return item.substring('soljson-'.length, item.length - '.js'.length);
 }
 
-const VERSION_LIST_URL = 'https://raw.githubusercontent.com/ethereum/solc-bin/gh-pages/bin/list.json';
+const VERSION_LIST_URL = 'https://solc-bin.ethereum.org/bin/list.json';
```

Comment:
Please note that every directory has its own list. If you're getting files from `emscripten-wasm32/` and `emscripten-asmjs/` now, it would be safer to fetch `emscripten-wasm32/list.json` and `emscripten-asmjs/list.json`. `bin/list.json` will work as long as all you need is the name because we're adding new releases to both `emscripten-wasm32/` and `bin/`  but would be a problem if you wanted to e.g. verify the keccak256 checksum listed in the JSON file (which will be different if the binary in `bin/` is the asm.js one).

By the way, here's a draft of the documentation update for solc-bin: https://github.com/ethereum/solidity/pull/10352. Any feedback is welcome in case anything is still missing.

Reply:
@cameel thanks for your valuable input! For now we're just using the `list.json` file to get the release versions and their commit hashes. I'll consider switching to fetching both of the lists you mentioned when I revisit the PR.